doc/tutorial/statistical_inference/finding_help.rst

```diff
@@ -13,14 +13,9 @@ ask on the `Mailing List <http://scikit-learn.org/stable/support.html>`_
 Q&A communities with Machine Learning practitioners
 ----------------------------------------------------
 
-  :Metaoptimize/QA:
+  :Stack Exchange:
 
-    A forum for Machine Learning, Natural Language Processing and
-    other Data Analytics discussions (similar to what Stackoverflow
-    is for developers): http://metaoptimize.com/qa
-
-    A good starting point is the discussion on `good freely available
-    textbooks on machine learning`_
+    http://meta.stackexchange.com/questions/130524/which-stack-exchange-website-for-machine-learning-and-computational-algorithms
```

Comment:
Metaoptimize doesn't seem to exist anymore so I was trying to offer another resource.